applications/nrf5340_audio/src/modules/audio_usb.c

```diff
@@ -196,33 +277,44 @@ int audio_usb_disable(void)
 {
 	int ret;
 
-	audio_usb_stop();
+	if (sample_usbd == NULL) {
+		LOG_ERR("USB device not initialized");
+		return -ENOTCONN;
+	}
 
-	ret = usb_disable();
+	ret = usbd_disable(sample_usbd);
 	if (ret) {
 		LOG_ERR("Failed to disable USB");
 		return ret;
 	}
 
+	audio_usb_stop();
+
 	return 0;
 }
 
-int audio_usb_init(void)
+int audio_usb_init(bool bidir)
```

Comment:
To be future compliant, this can have two bools. One for RX and one for TX so we can better scale for pure microphone scenarios.